dapps/templates/boilerplate/test/contract_spec.js

```diff
@@ -19,25 +19,23 @@ config({
   }
 }, (_err, web3_accounts) => {
   accounts = web3_accounts
-});
+});*/
 
 contract("SimpleStorage", function () {
-  this.timeout(0);
 
   it("should set constructor value", async function () {
-    let result = await SimpleStorage.methods.storedData().call();
-    assert.strictEqual(parseInt(result, 10), 100);
+    // let result = await SimpleStorage.methods.storedData().call();
```

Comment:
Instead of commenting out the code inside the `it` blocks, would it be better to use `.skip()`?